### PR TITLE
3djc/revise sdcard version

### PR DESCRIPTION
**Pull request by 3djc**
Needed file nuumber now displayed on error.
Server side generated sdcard file include version number too

### CODE REVIEW

**Comment by 3djc** [comment]
card version is kinda misleading, as some may think it is the card 'itself'

"Expects files version" ?

**Comment by 3djc** [comment]
"Expects files version" is too large for taranis with full version, so proposing "Expects version: "

**Comment by 3djc** [comment]
image

**Comment by kilrah** [comment]
That's not correct English... "Expected version" then

**Comment by schwabe** [comment]
Or maybe two lines?

Found version: xxxx
Expected version: yyyy

File: radio/src/opentx.cpp [hunk 1]
```diff
@@ -988,19 +988,21 @@ void checkSDVersion()
     FIL versionFile = {0};
     UINT read = 0;
     char version[sizeof(REQUIRED_SDCARD_VERSION)-1];
+    char error[sizeof(TR_WRONG_SDCARDVERSION)+ sizeof(version)];
 
+    strAppend(strAppend(error,STR_WRONG_SDCARDVERSION, sizeof(TR_WRONG_SDCARDVERSION)), REQUIRED_SDCARD_VERSION, sizeof(REQUIRED_SDCARD_VERSION));
```
Comment (bsongis) [inline radio/src/opentx.cpp]: one space after the comma?

**Pull request merged**